src/renderer/components/table/table-cell.tsx

```diff
@@ -5,6 +5,7 @@ import React, { ReactNode } from "react";
 import { autobind, cssNames, displayBooleans } from "../../utils";
 import { Icon } from "../icon";
 import { Checkbox } from "../checkbox";
+import { Resizable, ResizeCallback } from "re-resizable";
```

Comment:
We already have `ResizeAnchor` so I think we should use that.

Reply:
What do you think about to refactor the existing "table" (grid of divs)  to something more flexible like [this](https://autodesk.github.io/react-base-table/docs/get-started)

Reply:
I don't see how that particular example is "more flexible". If you look at the rendered DOM then you will see it too is a grid of divs.

Reply:
Yeah, it is, I mean It offers a more configurable and whole solution, it already contains a lot of features that we can use on the project. The current PR still contains some number of bugs that require global changes (like 2 scrollbars or getting calculated widths of columns), not sure that fixing these ones is easier than full refactoring of tables